gravity-forms/gw-gravity-forms-populate-form.php

```diff
@@ -75,8 +76,15 @@ public function prepare_form_for_population( $form ) {
 				$field['inputs'] = $inputs;
 			}
 
-			$field['inputName'] = $field['id'];
-
+			// If dynamic override is set, only map the field ID if it's empty
+			// Otherwise, leave it as is to populate from GF's dynamic population
+			if ( $this->_args['dynamic_override'] ) {
+				if ( strlen( $field['inputName'] ) < 1 ) {
```

Comment:
Curious, why not `empty` instead of `strlen`?

Reply:
@claygriffiths Old habit.  `empty('0')` is truthy. While I don't necessarily see people using `?param1=x&0=y&1=2` in the URL it's nice to know that they can 😅